app/util.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const request = require('request-promise');
+
+// returns a Date representing tomorrow at 12pm so that
+// we can issue darksky 'time machine' requests to get an
+// idea of the weather tomorrow at a time that should be
+// relevant to someone interested in the weather
+function getTomorrowAtNoon() {
+    let tomorrow = new Date();
+    tomorrow.setDate(tomorrow.getDate() + 1);
+    tomorrow.setHours(12);
+    return tomorrow;
+}
+
+// fixed coordinates used to query darksky
+const latitude = 38.8892681;
+const longitude = -77.0501425;
+const darkskyToken = process.env.DARKSKY_TOKEN || '';
+const darkskyUrlBase = `https://api.darksky.net/forecast/${darkskyToken}/${latitude},${longitude}`;
```

Comment:
Moved the latitude and longitude out. Ultimately, I'd probably want to mock the `require` module so that this doesn't have to be changed for testing purposes. Something like this: https://www.npmjs.com/package/request-promise#mocking-request-promise